metals/src/main/scala/scala/meta/metals/sbtserver/SbtServer.scala

```diff
@@ -48,11 +48,13 @@ object SbtServer extends LazyLogging {
   private def fail(message: String) = Task.now(Left(message))
 
   def readVersion(cwd: AbsolutePath): Option[String] = {
-    val path = cwd.resolve("project").resolve("build.properties")
-    val input = Files.newInputStream(path.toNIO)
     val props = new Properties()
-    Try { props.load(input) }
-    input.close()
+    val path = cwd.resolve("project").resolve("build.properties")
+    if (path.toFile.exists) {
+      val input = Files.newInputStream(path.toNIO)
+      Try { props.load(input) }
+      input.close()
```

Comment:
nitpick, `Try` returns a value you're then discarding. I feel this would be more "true to its spirit" if written as:

```scala
try {
  props.load(input)
} finally {
  input.close()
}
```

Reply:
`props.load(..)` returns `Unit`, so we actually want to ignore the result or any failure. Also I thought that `Try` is more Scala-idiomatic and these two snippets have the same outcome. But I can change it if you think it's clearer.

Reply:
I tend not to use `Try` in place of `try` because `Try` to me means that I'm interested in the value of a computation that can fail, which is not the case here.

Reply:
OK, I changed it